### PR TITLE
AzureKeyVault@1, @2: Fix incorrect inputs properties

### PR DESCRIPTION
**Pull request by 50Wliu**
**Task name**: AzureKeyVault@1, AzureKeyVault@2

**Description**: The `inputs` JSON in `task.json` was using `options` instead of `properties` for the SecretsFilter & RunAsPreJob inputs, leading to incorrect validation downstream in the azure-pipelines-vscode extension.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) Fixes https://github.com/microsoft/azure-pipelines-vscode/issues/410.

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

This is input validation, so I'm unsure how to test it as I don't know how the schema gets generated.

### CODE REVIEW

File: Tasks/AzureKeyVaultV2/task.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 189,
+        "Minor": 192,
```
Comment (KonstantinTyukalov) [inline Tasks/AzureKeyVaultV2/task.json:17]: @50Wliu  Could you please bump the minor version of the task? 
Also, seems that changes from task.lock.json file is missing. Could you please update it as well?
 You can do this following this guide: https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md

**Comment by KonstantinTyukalov** [comment]
@rohit-batra, @rvairavelu - could you please take a look at this changes?

File: Tasks/AzureKeyVaultV2/task.json (hunk 1 above)
Comment (50Wliu) [reply Tasks/AzureKeyVaultV2/task.json]: Sure - just updated all of them to minor version 199.

File: Tasks/AzureKeyVaultV1/task.loc.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 198,
+    "Minor": 199,
```
Comment (rohit-batra) [inline Tasks/AzureKeyVaultV1/task.loc.json:17]: should be 200 now

File: Tasks/AzureKeyVaultV2/task.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 189,
+        "Minor": 199,
```
Comment (rohit-batra) [inline Tasks/AzureKeyVaultV2/task.json:17]: should be 200 now

**Review by rohit-batra — APPROVED** [review]

**Comment by 50Wliu** [comment]
Yay, thank you!

**Pull request merged**